testdata/list-clouds.md

```diff
@@ -0,0 +1,23 @@
+# Command "juju clouds"
+
+> See also: [add-cloud](https://docs.com/add-cloud), [update-cloud](https://docs.com/update-cloud), [remove-cloud](https://docs.com/remove-cloud), [update-credential](https://docs.com/update-credential)
```

Comment:
The docs.com link looks wrong... (thought I'm guessing that is so because this is just a test).

Reply:
Yes it's because it's just a test.